webrtc.html

```diff
@@ -7932,8 +7932,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 <li>
                   <p>If <var>channel</var> was the first
                   <code><a>RTCDataChannel</a></code> created on
-                  <var>connection</var>, <a>update the negotiation-needed
-                  flag</a> for <var>connection</var>.</p>
+                  <var>connection</var>, queue a task to <a>update the
```

Comment:
This algorithm will already queue a task to fire the event, so why is this necessary? So that interactions with state and internal slots don't happen "in parallel"? I think we could just move this up before the "in parallel" part then.

Reply:
By which I mean, make the order:

1. Return channel
2. Update negotiation-needed flag
3. Run steps to set up the underlying SCTP association in parallel

Reply:
If we return in step 1. above, I guess we need to wait for stable state before we can continue and do step 2. on the main loop and then do step 3 in parallel. To avoid that I think we could do 2, 1, 3.